SwrveSDK/SDK/Track/Swrve.m

```diff
@@ -1950,13 +1966,21 @@ - (UInt64) getInstallTime:(NSString*)fileName withSecondaryFile:(NSString*)secon
     if (seconds > 0)
     {
         UInt64 result = seconds;
-        return result * 1000;
+        return result;
     }
 
-    UInt64 time = [self getTime];
-    NSString* currentTime = [NSString stringWithFormat:@"%llu", time/(UInt64)1000L];
+    UInt64 time = [self secondsSinceEpoch];
+    NSString* currentTime = [NSString stringWithFormat:@"%llu", time];
     [currentTime writeToFile:fileName atomically:YES encoding:NSUTF8StringEncoding error:nil];
-    return (time / 1000 * 1000);
+    return time;
+}
+
+- (UInt64) secondsSinceEpoch {
+    return (unsigned long long)([[NSDate date] timeIntervalSince1970]);
+}
+
+- (UInt64) millisecondsSinceEpoch {
```

Comment:
actually not currently. I was putting it in to provide an option but I'll take it out for now.